### PR TITLE
chore(Examples): Test examples on Rev. B production silicon for MAX32655

### PR DESCRIPTION
**Pull request by Jacob-Scheiffler**
This PR includes the changes related to testing, debugging, and updating each of the non-BLE examples for the MAX32655 Rev. B production parts. The Flash and I2C_Playback examples each have minor bug fixes and the changes for the rest of the examples are mainly just README updates.

### CODE REVIEW

File: Examples/MAX32655/Coremark/README.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ Universal instructions on building, flashing, and debugging this project can be
 
 ### Project-Specific Build Notes
 
-To comply with the CoreMark rules, the only source files which are included in this example directory are the core_portme.c/.h files, the rest (including main) are located in the [Coremark](../../../Libraries/Coremark/) library.
+* To comply with the CoreMark rules, the only source files which are included in this example directory are the core_portme.c/.h files, the rest (including main) are located in the [Coremark](../../../Libraries/Coremark/) library.
```
Comment (sihyung-maxim) [inline Examples/MAX32655/Coremark/README.md:17]: Very minor, but I think it should be `CoreMark` (it's a registered trademark). We goofed when we added the library and named it `Coremark`. We can't change the directory name, but we can start naming it properly going forward.

**Review by sihyung-maxim — APPROVED** [review]

**Comment by sihyung-maxim** [comment]
Also, I recommend merging the latest changes from main so you have the auto-labeler updates.

File: Examples/MAX32655/Coremark/README.md (hunk 1 above)
Comment (Jacob-Scheiffler) [reply Examples/MAX32655/Coremark/README.md:17]: I made a note to make that change. Since it affects multiple parts, I'll do that in a separate PR.

**Review by Jake-Carter — APPROVED** [review]
bueno

**Pull request merged**